src/main.cpp

```diff
@@ -116,7 +116,7 @@ int main(int argc, char *argv[])
 	
 	QCoreApplication::setOrganizationName("ProMesh");
     QCoreApplication::setOrganizationDomain("promesh3d.com");
-    QCoreApplication::setApplicationName("ProMesh4.3.10");
+    QCoreApplication::setApplicationName("ProMesh4.3.18");
```

Comment:
While 4.3.18 is more accurate, changing this string will create a new file where the QSettings contents are stored. When a user opens ProMesh with the changed application name, old settings will be gone. Currently this only affects the file open/save path. But in the future it would also apply to the custom script paths, if you will adjust their storage to QSettings.
Maybe you could simply change it to "ProMesh"?